substrate-processor/src/interfaces/gateway.ts

```diff
@@ -9,7 +9,7 @@ export interface BatchRequest {
     events?: ObjectRequest[]
     calls?: ObjectRequest[]
     evmLogs?: EvmLogRequest[]
-    ethereumTransactions?: EthereumTransactionRequest[]
+    ethereumTransactCalls?: EthereumTransactRequest[]
```

Comment:
I think we should stick to one name `ethereumTransactions` or `ethereumTransactCalls`, overwise there can be confusion due different names in different parts of code

Reply:
oh, was fixed in commite above